opennms-doc/guide-admin/src/asciidoc/text/sentinel/sentinel.adoc

```diff
@@ -374,3 +387,132 @@ Simply copy it to `${SENTINEL_HOME}/deploy/`.
     </feature>
 </features>
 -----
+
+=== Persisting Collection Sets to Newts
+
+In the previous chapter it is described on how to setup _{opennms-product-name}_, _Minion_ and _Sentinel_ in order to distribute the procession of flows.
+However, it only covered flow processing adapters, but there are more, e.g. the `NxosGpbAdapter`, which can also be run on a _Sentinel_.
+
+==== Adapters
+
+This chapter describes the various adapters which may contain sample data which may be stored to a Persistence Storage and can also run on a _Sentinel_.
+At the moment only _Newts_ is supported as a Persistence Storage.
+See chapter <<ga-sentinel-configure-newts>> on how to configure _Newts_.
+
+In order to get it to work properly, please note, that an apropriate listener on the _Minion_ must also be configured.
+The name of the listener should share the same name on _Sentinel_.
+
+===== SFlowTelemetryAdapter
+
+In order to use this adapter, the feature `sentinel-flows` and `sentinel-newts` must be installed.
+In addition either `sentinel-jms` or `sentinel-kafka` should be installed and configured properly.
+See the previous _Flow Procession_ chapter for more details.
+
+If only sample data should be persisted, the following commands can be run on the _Sentinel_'s Karaf Shell
+
+----
+$ ssh -p 8301 admin@localhost
+----
+
+----
+admin@sentinel> config:edit org.opennms.features.telemetry.adapters-sflow
+admin@sentinel> config:property-set name SFlow-Telemetry
+admin@sentinel> config:property-set class-name org.opennms.netmgt.telemetry.adapters.netflow.sflow.SFlowTelemetryAdapter
+admin@sentinel> config:property-set parameters.script  /opt/sentinel/etc/sflow-host.groovy
+admin@sentinel> config:update
+----
+
+If SFlow flows and the sample data should be processed, multiple adapters can be configured:
+
+----
+config:edit org.opennms.features.telemetry.adapters-sflow-telemetry
+config:property-set name SFlow
+config:property-set adapters.1.name SFlow-Adapter
+config:property-set adapters.1.class-name org.opennms.netmgt.telemetry.adapters.netflow.sflow.SFlowAdapter
+config:property-set adapters.2.name SFlow-Telemetry
+config:property-set adapters.2.class-name org.opennms.netmgt.telemetry.adapters.netflow.sflow.SFlowTelemetryAdapter
+config:property-set adapters.2.parameters.script /opt/sentinel/etc/sflow-host.groovy
+config:update
+----
+
+Please note, that in both cases the file `/opt/sentinel/etc/sflow-host.groofy` must be provided manually, e.g. by manually copying it over from _{opennms-product-name}_.
+
+===== NxosGpbAdapter
+
+In order to use this adapter, the feature `sentinel-telemetry-nxos` and `sentinel-newts` must be installed.
+In addition either `sentinel-jms` or `sentinel-kafka` should be installed and configured properly.
+See the previous _Flow Procession_ chapter for more details.
+
+Besides this, configuration files from _{opennms-product-name}_ must be copied to _Sentinel_ to `/opt/sentinel/etc`.
```

Comment:
It looks like the `OpenNMS :: Features :: Distributed :: Collection` bundle requires these datacollection files to be in place, irrespective of which adapter is being used. Perhaps we should restructure the docs to reflect this?

Reply:
The config files are only required by the `NxosGpbAdapter` and `JtiGpbAdapter`. At the moment the `SFlowTelemetryAdapter` does not require them, thus the structure of the documentation.

However we can make it always require those files, and can re-structure the docs a bit.

Reply:
The bundle doesn't load without those configuration files in place. Doesn't the `SFlowTelemetryAdapter` require that bundle to be loaded in order to persist the collection sets?

Reply:
Yes, the bundle requires those configuration files in place, however the `SFlowTelemetryAdapter` does not require that bundle.

The system test is also reflecting that, as it does not load the bundle, but the tests still succeed.